portal-api/src/v1/controllers/deal-status.controller.js

```diff
@@ -66,6 +66,8 @@ exports.update = async (req, res) => {
 
     const currentStatus = deal.status;
 
+        console.info('Updating portal deal %s status to %s from %s', dealId, newStatus, currentStatus);
```

Comment:
fwiw - over indented?

Reply:
For troubleshooting with race condition.